### PR TITLE
Update: updated German translation de_DE

### PR DESCRIPTION
**Pull request by lighthousebulb**
I just like to help the Project, so I updated some translation strings

### CODE REVIEW

File: CliClient/locales/de_DE.po [hunk 1]
```diff
@@ -1357,27 +1361,32 @@ msgstr "ID"
...
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:181
-#, fuzzy
 msgid "Re-encrypt data"
-msgstr "Datum auswählen"
+msgstr "Daten neu verschlüsseln"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:183
+#, fuzzy
 msgid ""
 "The default encryption method has been changed to a more secure one and it "
```
Comment (tessus) [inline CliClient/locales/de_DE.po]: Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.

File: CliClient/locales/de_DE.po [hunk 2]
```diff
@@ -1704,8 +1728,9 @@ msgid "In: %s"
 msgstr "In: %s"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/NoteText.min.js:1661
+#, fuzzy
 msgid "Front"
-msgstr ""
+msgstr "Vorderseite"
```
Comment (tessus) [inline CliClient/locales/de_DE.po]: Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.

File: CliClient/locales/de_DE.po [hunk 3]
```diff
@@ -1357,27 +1361,32 @@ msgstr "ID"
...
 msgstr ""
+"Sie können das folgende Werkzeug  verwenden, um Ihre Daten erneut zu "
+"verschlüsseln, falls einige Ihrer Notizen mit einer veralteten "
+"Verschlüsselungsmethode verschlüsselt sein sollten."
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:185
-#, javascript-format
+#, fuzzy, javascript-format
 msgid ""
 "In order to do so, your entire data set will have to be encrypted and "
```
Comment (tessus) [inline CliClient/locales/de_DE.po]: Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.

**Comment by lighthousebulb** [comment]
Well, I installed Poedit as ist is. (Version 2.3 5890). 
I did not add die `fuzz` Flag intentionally. It just happends that Poedit did this, when I maked something as "Needs Work".
Maybe there is an Option in Poedit to switch the Flag from `fuzzy`to `Needs Work`?

The interesting part ist, that the actual de_DE.po file I downloaded from the repository, already containes `fuzzy` Flags. So I don't get the point...

**Comment by tessus** [comment]
> It just happends that Poedit did this, when I maked something as "Needs Work".

Right, if you mark something as `Needs Work`, it means the translation can't be used, because it might be wrong.

> The interesting part ist, that the actual de_DE.po file I downloaded from the repository, already containes fuzzy Flags. So I don't get the point...

We use gettext to extract the translatable strings from the source code. Certain phrases might be similar to others and it adds a translation which is automatically marked as fuzzy, because the gettext program wouldn't know, if it's a correct translation.

**Comment by tessus** [comment]
`Needs Work` sets the `fuzzy` flag in the `.po` file. They are interchangeable: if you mark something in the `.po` file as fuzzy, it shows up as `Needs Work` in Poedit, and vice versa.

**Comment by lighthousebulb** [comment]
Ah, ok, I got you. Yes I marked some of the translations as `fuzzy` to point someone else to my translation suggestion. I'm aware that you might not use the suggestions so far.
At first I thought you might not use my provided PO-File at all.
So all is as intended...

**Comment by tessus** [comment]
> I'm aware that you might not use the suggestions so far.

It's not me who's not using the translation, but the app. As soon as something is marked as fuzzy, the translation is not used.

So, if you really meant the translations only to be a starting point for somebody else, all is good.
I just thought you marked them by accident.

**Comment by tessus** [comment]
Ooops, it's way past my bed time. Heading to bed....
